src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -847,7 +847,7 @@ void PolicyHandler::OnPendingPermissionChange(
         app,
         mobile_apis::HMILevel::HMI_NONE,
         mobile_apis::AudioStreamingState::NOT_AUDIBLE,
-        true);
+        false);
```

Comment:
@LuxoftAKutsan 
Just talked to BA team - seems requirements still not finalized. Please check with T.Melnyk.